addon/authenticators/torii.js

```diff
@@ -32,11 +32,16 @@ export default Base.extend({
     @public
   */
   restore(data) {
+    let torii = this.get('torii');
+    if (Ember.isEmpty(torii)) {
+      Ember.Logger.warn('You are trying to use `torii-authenticator` but torii is not installed. Install torii using `ember install torii`.');
+    }
```

Comment:
Can you extract this into its own method (maybe `_assertToriiIsPresent`) as it's exactly the same as below?

Also this should use `Ember.assert` instead of checking and then logging.